onnxruntime/contrib_ops/cpu/bert/embed_layer_norm.cc

```diff
@@ -98,13 +98,15 @@ Status EmbedLayerNorm<T>::Compute(OpKernelContext* context) const {
   }
 
   // Calculate mask
-  {
+  if (nullptr != mask) {
     const int* mask_data = mask->template Data<int>();
     for (int b = 0; b < batch_size; b++) {
       mask_index->template MutableData<int>()[b] = static_cast<int>(std::count_if(mask_data + (b * sequence_length),
                                                                                   mask_data + (b * sequence_length) + sequence_length,
                                                                                   [](int v) { return v == 1; }));
     }
+  } else {
+    memset(mask_index->template MutableData<int>(), 0, batch_size * sizeof(int));
```

Comment:
>int [](start = 44, length = 3)

int is not a datatype supported in Tensors. It has a different size on different platforms. On WIndows it is 32 bit but on Linux it is 64-bit. Which one would you like to use?

Reply:
Good catch. I'll update to int32_t.